client/src/Components/Organization/OrganizationCard.jsx

```diff
@@ -35,8 +36,15 @@ const useStyles = makeStyles(theme => ({
   }
 }));
 
+
 export default function ImgMediaCard() {
   const classes = useStyles();
+  const history = useHistory();
+
+  function handleChange(){
+    history.push('./OrganizationDetail');
```

Comment:
same here

Reply:
Thanks Rupesh for the suggestions. I have made the required changes.